diff --git a/src/config.ts b/src/config.ts
index 6937747..4444600 100644
--- a/src/config.ts
+++ b/src/config.ts
@@ -202,11 +202,11 @@ export const CONFIG: Config = {
   nodeExternalIpForRemoteLocalNetwork: '127.0.0.1',
   collectorSourcing: {
     enabled: true,
-    collectorApiServerUrl: 'http://0.0.0.0:6001',
+    collectorApiServerUrl: 'http://0.0.0.0:7001',
   },
   serviceValidatorSourcing: {
     enabled: false,
-    serviceValidatorUrl: 'http://0.0.0.0:7001',
+    serviceValidatorUrl: 'http://0.0.0.0:9001',
   },
 
   ServicePointsPerSecond: 200,
@@ -214,7 +214,7 @@ export const CONFIG: Config = {
   ServicePoints: {
     ['aalg-warmup']: 20,
   },
-  enableBlockCache: false,
+  enableBlockCache: true,
   useRoundRobinConsensorSelection: true,
   devPublicKeys: {
     // '': DevSecurityLevel.Unauthorized,
