diff --git a/src/config.ts b/src/config.ts
index e66e445..08898f5 100644
--- a/src/config.ts
+++ b/src/config.ts
@@ -180,11 +180,11 @@ export const CONFIG: Config = {
   isRemoteLocalNetwork: false,
   nodeExternalIpForRemoteLocalNetwork: '127.0.0.1',
   collectorSourcing: {
-    enabled: false,
+    enabled: true,
     collectorApiServerUrl: 'http://0.0.0.0:6001',
   },
   serviceValidatorSourcing: {
-    enabled: false,
+    enabled: true,
     serviceValidatorUrl: 'http://0.0.0.0:9001',
   },
 
@@ -193,7 +193,7 @@ export const CONFIG: Config = {
   ServicePoints: {
     ['aalg-warmup']: 20,
   },
-  enableBlockCache: false,
+  enableBlockCache: true,
   useRoundRobinConsensorSelection: true,
   devPublicKeys: {
     // '': DevSecurityLevel.Unauthorized,
