diff --git a/src/config.ts b/src/config.ts
index db6061c..3862259 100644
--- a/src/config.ts
+++ b/src/config.ts
@@ -183,11 +183,11 @@ export const CONFIG: Config = {
   isRemoteLocalNetwork: false,
   nodeExternalIpForRemoteLocalNetwork: '127.0.0.1',
   collectorSourcing: {
-    enabled: false,
+    enabled: true,
     collectorApiServerUrl: 'http://0.0.0.0:6001',
   },
   serviceValidatorSourcing: {
-    enabled: false,
+    enabled: true,
     serviceValidatorUrl: 'http://0.0.0.0:9001',
   },
 
@@ -196,6 +196,6 @@ export const CONFIG: Config = {
   ServicePoints: {
     ['aalg-warmup']: 20,
   },
-  enableBlockCache: false,
+  enableBlockCache: true,
   useRoundRobinConsensorSelection: true,
 }
