diff --git a/src/config.ts b/src/config.ts
index a633cf3..8c3df19 100644
--- a/src/config.ts
+++ b/src/config.ts
@@ -135,8 +135,8 @@ export const CONFIG: Config = {
   explorerUrl: process.env.EXPLORER_URL || 'http://127.0.0.1:6001',
   queryFromExplorer: false,
   generateTxTimestamp: true,
-  nodelistRefreshInterval: Number(process.env.NODELIST_REFRESH_INTERVAL) || 30000,
-  defaultRequestRetry: 5,
+  nodelistRefreshInterval: 5000,
+  defaultRequestRetry: 10,
   gasEstimateMethod: process.env.GAS_ESTIMATE_METHOD || 'serviceValidator', //serviceValidator or replayEngine or validator
   gasEstimateInvalidationIntervalInMs: 1000 * 60 * 60 * 2, // 2 hours
   gasEstimateUseCache: false,
